packages/app-headless-cms/src/modules/pageTemplates/views/list/CloneContentModelDialog.tsx

```diff
@@ -0,0 +1,283 @@
+import React, { useState, useCallback, useEffect } from "react";
+import { css } from "emotion";
+import get from "lodash/get";
+import { useRouter } from "@webiny/react-router";
+import { Form } from "@webiny/form";
+import { Input } from "@webiny/ui/Input";
+import { Select } from "@webiny/ui/Select";
+import { useSnackbar } from "@webiny/app-admin/hooks/useSnackbar";
+import { CircularProgress } from "@webiny/ui/Progress";
+import { validation } from "@webiny/validation";
+import { useMutation, useQueryLocale } from "~/admin/hooks";
+import { i18n } from "@webiny/app/i18n";
+import { ButtonDefault } from "@webiny/ui/Button";
+import * as UID from "@webiny/ui/Dialog";
+import { Grid, Cell } from "@webiny/ui/Grid";
+import { addModelToGroupCache, addModelToListCache } from "./cache";
+import { CmsEditorContentModel, CmsModel } from "~/types";
+import { useI18N } from "@webiny/app-i18n/hooks/useI18N";
+import {
+    CREATE_CONTENT_MODEL_FROM,
+    LIST_MENU_CONTENT_GROUPS_MODELS,
+    CreateCmsModelFromMutationResponse,
+    CreateCmsModelFromMutationVariables,
+    ListMenuCmsGroupsQueryResponse
+} from "~/admin/viewsGraphql";
+import { CmsGroup } from "~/admin/views/contentModelGroups/graphql";
+import { CmsGroupOption } from "~/admin/views/contentModels/types";
+
+const t = i18n.ns("app-headless-cms/admin/views/content-models/clone-content-model-dialog");
+
+const narrowDialog = css`
+    .mdc-dialog__surface {
+        width: 600px;
+        min-width: 600px;
+    }
+`;
+
+const noPadding = css`
+    padding: 5px !important;
+`;
+
+export interface Props {
```

Comment:
If this file is going to be re-exported at some point, and there are other files that export `Props` interface, it will cause problems. Maybe use `CloneContentModelDialogProps`.